homework.js

```diff
@@ -0,0 +1,118 @@
+//1 
+let arr = [1, 2, 3, 4, 5];
+function sum(arr) {
+    var s = 0;
+    for (i = 0; i < arr.length; i++) {
+       s += arr[i];
+    }
+    return s
+  }
+  
+  //2
+let numStr = [1, 2, 3, 4, 5];
+
+function sumEvens(numStr){
```

Comment:
the input is a number not an array

Reply:
let arr = [1, 2, 3, 4, 5];
function summation() {
    var s = 0;
    for (i = 0; i < arr.length; i++) {
       s += arr[i];
    }
    return s
  } 
// call it by 
summation(5);